src/words/sedbased.jl

```diff
@@ -124,7 +131,22 @@ Depends exactly what you want it for.
 
 This matches to the most commonly used `nltk.word_tokenize`, minus the sentence tokenizing step.
 """
-@generated function nltk_word_tokenize(input::AbstractString)
-    script = joinpath(@__DIR__, "nltk_word.sed")
-    generate_tokenizer_from_sed(script, true)
+function nltk_word_tokenize(input)
+  ts = TokenBuffer(input)
+  isempty(input) && return ts.tokens
+  stop = ts.input[end] == '.'
```

Comment:
ending periods interf
```suggestion
  stop = ts.input[end] == '.'  # Ending stops interfere with the other rules so remove it and put it back later
```